public/kibi_timeline_vis_controller.js

```diff
@@ -95,11 +95,12 @@ define(function (require) {
                 params: {
                   //kibi params
                   // .path property doesn't exist in case of multi-fields
-                  labelFieldSequence: fields[i].byName[group.labelField].path.length === 0
+                  labelFieldSequence: !fields[i].byName[group.labelField].path.length
                     ? [ group.labelField ] : fields[i].byName[group.labelField].path,
-                  startFieldSequence: fields[i].byName[group.startField].path.length === 0
+                  startFieldSequence: !fields[i].byName[group.startField].path.length
```

Comment:
Use only `fields[i].byName[group.startField].path` here. It is used to differentiate between Kibi and Kibana in timeline helper class methods.